action-api/src/test/java/com/powsybl/action/EqualsActionTest.java

```diff
@@ -0,0 +1,472 @@
+                .withTapPosition(2)
+                .withRelativeValue(true)
+                .build();
+        new EqualsTester()
+                .addEqualityGroup(action1, action2)
+                .addEqualityGroup(action3)
+                .addEqualityGroup(action4)
+                .addEqualityGroup(action5)
+                .testEquals();
+    }
+
+    @Test
+    void generatorAction() {
+        GeneratorAction action1 = new GeneratorActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withActivePowerValue(100)
+                .withActivePowerRelativeValue(false)
+                .withTargetQ(50)
+                .build();
+        GeneratorAction action2 = new GeneratorActionBuilder().withId("id")
+                .withGeneratorId("neId")
+                .withActivePowerValue(100)
+                .withActivePowerRelativeValue(false)
+                .withTargetQ(50)
+                .build();
+        GeneratorAction action3 = new GeneratorActionBuilder().withId("id2")
+                .withNetworkElementId("neId")
+                .withActivePowerValue(100)
+                .withActivePowerRelativeValue(true)
+                .withTargetQ(50)
+                .build();
+        GeneratorAction action4 = new GeneratorActionBuilder().withId("id")
+                .withNetworkElementId("neId2")
+                .withActivePowerValue(100)
+                .withActivePowerRelativeValue(false)
+                .withVoltageRegulatorOn(true)
+                .withTargetQ(50)
+                .build();
+        GeneratorAction action5 = new GeneratorActionBuilder().withId("id2")
+                .withNetworkElementId("neId")
+                .withActivePowerValue(100)
+                .withActivePowerRelativeValue(false)
+                .withTargetV(50)
+                .build();
+        new EqualsTester()
+                .addEqualityGroup(action1, action2)
+                .addEqualityGroup(action3)
+                .addEqualityGroup(action4)
+                .addEqualityGroup(action5)
+                .testEquals();
+    }
+
+    @Test
+    void hvdcAction() {
+        HvdcAction action1 = new HvdcActionBuilder().withId("id")
+                .withNetworkElementId("neId")
+                .withAcEmulationEnabled(true)
+                .build();
+        HvdcAction action2 = new HvdcActionBuilder().withId("id")
+                .withHvdcId("neId")
+                .withAcEmulationEnabled(true)
+                .build();
+        HvdcAction action3 = new HvdcActionBuilder().withId("id2")
+                .withNetworkElementId("neId")
+                .withDroop(2.)
+                .build();
+        HvdcAction action4 = new HvdcActionBuilder().withId("id")
+                .withNetworkElementId("neId2")
+                .withAcEmulationEnabled(true)
+                .withP0(2.)
+                .build();
+        HvdcAction action5 = new HvdcActionBuilder().withId("id")
```

Comment:
To fully cover the `equals()` method, could you add actions with changes on the following attributes:
- activePowerSetpoint
- converterMode
- droop
- p0